### PR TITLE
docs: refine CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by emptinesssubodhi**
1. More concise and professional
2. fix invalid discord link

### CODE REVIEW

File: CONTRIBUTING-zh.md
Comment (jolestar) [inline CONTRIBUTING-zh.md:1]: Please keep the Chinese version of the contributing document. @geometryolife can help to sync the changes to Chinese.

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ To get started with contributing:
 
 ## Community
 
-* [Discord](https://discord.gg/rooch)
+* [Discord](https://discord.gg/hpJZkBFggN)
```
Comment (geometryolife) [inline README.md:99]: Is this link permanently valid?

File: CONTRIBUTING_zh.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+* `Summary` - 描述构成该 Pull Request 的内容，以及您对代码所做的更改。例如，修复了哪个问题。
+
+## 问题
+
+Rooch 使用 [GitHub issues](https://github.com/rooch-network/rooch/issues) 跟踪错误。请包含必要的信息和说明，以便重现您的问题。
+
+## 文档
+
+所有开发者文档均发布在 Rooch 开发者网站 [rooch.network](https://rooch.network/)。
```
Comment (geometryolife) [inline CONTRIBUTING_zh.md:69]: Go to the `developer` page here or `docs` page

**Review by geometryolife — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jolestar) [reply README.md:99]: I add more boost to the discord server, now the https://discord.gg/rooch link is working.

**Review by geometryolife — APPROVED** [review]

**Pull request merged**